documentation/svfsiplus/theory/fluid/readme.md

```diff
@@ -0,0 +1,258 @@
+
+$$
+w_{i} = \sum_{a=1}^{n_{en}} N_{a}^{w}w_{ai},
+$$
+
+$$
+q = \sum_{a=1}^{n_{en}} N_{a}^{q}q_{a},
+$$
+
+where $N_{a}^{w}$ and $N_{a}^{q}$ are the nodal shape (basis) functions for the velocity and pressure spaces, respectively, and $w_{ai}$ and $q_{a}$ are the associated arbitrary nodal coefficients. Similarly, the trial functions are represented by
+
+$$
+u_{i} = \sum_{a=1}^{n_{en}} N_{a}^{w}u_{ai},
+$$
+
+$$
+p = \sum_{a=1}^{n_{en}} N_{a}^{q}p_{a}.
+$$
+
+We then multiply the Navier-Stokes-Brinkman equations by $\boldsymbol{w}$ and $q$, respectively, and integrate by parts to obtain the standard Galerkin momentum and continuity weak forms <a href="#ref-7">[7]</a> <a href="#ref-5">[5]</a>,
+
+$$
+\int \rho w_{i}\frac{du_{i}}{dt} \,d\Omega + \int \rho w_{i}u_{k}u_{i, k} \,d\Omega + \int w_{i, j}\sigma_{ij} \,d\Omega + \int \frac{\mu}{K}w_{i}u_{i} \,d\Omega - \int w_{i}\rho b_{i} \,d\Omega - \int_{\Gamma_{h}} w_{i}h_{i} \,d\Gamma = 0,
+$$
+
+$$
+\int qu_{i,i} \,d\Omega = 0.
+$$
+
+These two equations can be added together to obtain
+
+$$
+\int qu_{i,i} \,d\Omega + \int \rho w_{i}\frac{du_{i}}{dt} \,d\Omega + \int \rho w_{i}u_{k}u_{i, k} \,d\Omega + \int w_{i, j}\sigma_{ij} \,d\Omega + \int \frac{\mu}{K}w_{i}u_{i} \,d\Omega - \int w_{i}\rho b_{i} \,d\Omega - \int_{\Gamma_{h}} w_{i}h_{i} \,d\Gamma = 0.
+$$
+
+### Stabilized weak form
+
+The standard weak form is generally not stable. Additional terms must be added to stabilize it. We will apply the residual-based variational multiscale (RBVMS / VMS) method for stabilization <a href="#ref-1">[1]</a> <a href="#ref-5">[5]</a>. 
+
+In VMS, the velocity and pressure terms are separated into coarse-scale and fine-scale components, such that
+
+$$
+\boldsymbol{u} = \boldsymbol{u}^{h} + \boldsymbol{u}', \\
+$$
+
+$$
+p = p^{h} + p',
+$$
+
+where the $h$-superscript designates the coarse-scale components and the $'$-superscript denotes the fine-scale components. The fine-scale terms are defined as
+
+$$
+\boldsymbol{u}' = -\frac{\tau_{SUPS}}{\rho}\boldsymbol{r}_{M}\left(\boldsymbol{u}^{h}, p^{h}\right),
+$$
+
+$$
+p' = -\rho\nu_{LSIC}r_{C}\left(\boldsymbol{u}^{h}\right),
+$$
+
+where the PDE residuals are
+
+$$
+\boldsymbol{r}_{M}\left(\boldsymbol{u}^{h}, p^{h}\right) = \rho\left(\frac{d\boldsymbol{u}^{h}}{dt} + \boldsymbol{u}^{h} \cdot \boldsymbol{\nabla} \boldsymbol{u}^{h} - \boldsymbol{b}\right) - \boldsymbol{\nabla} \cdot \boldsymbol{\sigma}^{h} + \frac{\mu}{K}\boldsymbol{u}^{h},
+$$
+
+$$
+r_{C}\left(\boldsymbol{u}^{h}\right) = \boldsymbol{\nabla} \cdot \boldsymbol{u}^{h}.
+$$
+
+The stabilization parameters are defined as
+
+$$
+\tau_{SUPS} = \tau_{M} = \left(\frac{4}{\Delta t^{2}} + \boldsymbol{u}^{h} \cdot \boldsymbol{G}\boldsymbol{u}^{h} + C_{1}\nu^{2}\boldsymbol{G}:\boldsymbol{G} + \left(\frac{\nu}{K}\right)^{2}\right)^{-1/2},
```

Comment:
@JonathanPham do you think we should define G as the metric tensor or is it too much detail?

Reply:
Just added that and a reference too.